tools/dockerfile/distribtest/ruby_centos7_x64/Dockerfile

```diff
@@ -14,6 +14,24 @@
 
 FROM centos:7
 
-RUN yum install -y ruby
+RUN yum update
```

Comment:
btw, in dockerfiles  "yum update" and "yum install" as separate commands are considered a bad practice (because it makes caching behave weirdly)

Reply:
done, makes sense. folded this update into the single commit